.travis.yml

```diff
@@ -9,4 +9,11 @@ compiler:
 # temporary hack with sed until we will have a proper configure script
 script:
```

Comment:
You may want to insert ` - set -e` here, to ensure the script aborts on the first error